backend/config/blockexplorer.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+// BlockExplorer defines a selectable block explorer.
+type BlockExplorer struct {
+	Name string `json:"name"`
+	Url  string `json:"url"`
```

Comment:
document the fields as well. Especially that the Url is a prefix and the txID will be appended.